tools/fireacme/99-firehose-acme.sh

```diff
@@ -6,7 +6,7 @@ export PATH=$PATH:/app
 
```

Comment:
I realize this file was never named correctly, can you renamed `tools/fireacme/99-firehose-arweave.sh`

Reply:
And ensure you search for its usage also.